codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/autoconstructcontainers/listofblobstypecopier.java

```diff
@@ -0,0 +1,25 @@
+package software.amazon.awssdk.services.jsonprotocoltests.model;
+
+import static java.util.stream.Collectors.toList;
+
+import java.nio.ByteBuffer;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Generated;
+import software.amazon.awssdk.core.runtime.StandardMemberCopier;
+import software.amazon.awssdk.core.util.DefaultSdkAutoConstructAwareList;
+
+@Generated("software.amazon.awssdk:codegen")
+final class ListOfBlobsTypeCopier {
+    static List<ByteBuffer> copy(Collection<ByteBuffer> listOfBlobsTypeParam) {
+        if (listOfBlobsTypeParam == null) {
```

Comment:
Do we need to check auto construct here to? Can we add a test to make sure that a POJO with an auto constructed list is preserved when we go from toBuilder and back?

Reply:
Hmm, we shouldn't have to. We use the model copiers on all builder setters (unless the param is something immutable like `String`) and the builder constructor from the model class, so it should always be type of `AutonConstructAwareList`, if that's what the customization config says.

Would making the field type `AutoConstructAwareList` suffice?

Reply:
I just realized that we wrap in `unmodifiableList` before returning which will mess with `instanceof` checks we might do later in the marshallers. I'll need to make a few changes.

Reply:
Updated to do an instance check. Also added tests to verify this behavior